python/ray/tests/test_dynres.py

```diff
@@ -172,7 +189,8 @@ def set_res(resource_name, resource_capacity, res_client_id):
             target_node = next(
                 node for node in ray.nodes() if node["NodeID"] == nid)
             resources = target_node["Resources"]
-            resources_created.append(resources[res_name] == res_capacity)
+            resources_created.append(
+                resources.get(res_name, None) == res_capacity)
         success = all(resources_created)
     assert success
 
```

Comment:
This is similar to the `wait_for_condition()` functionality, should we use that here? Something like:

```
def check_resources():
    resources_created = []
        for nid in target_node_ids:
            target_node = next(
                node for node in ray.nodes() if node["NodeID"] == nid)
            resources = target_node["Resources"]
            resources_created.append(resources.get(res_name, None) == res_capacity)
        success = all(resources_created)
    return success
wait_for_condition(check_resources)
```

Reply:
thanks, fixed.